test/csuite/random_abort/main.c

```diff
@@ -814,8 +814,9 @@ main(int argc, char *argv[])
     if (chdir(home) != 0)
         testutil_die(errno, "parent chdir: %s", home);
 
-    /* Copy the data to the home directory for debugging purposes given its absolute path */
-    testutil_copy_data();
+    if (!verify_only)
+        /* Copy the data to the home directory for debugging purposes given its absolute path */
```

Comment:
NP: this comment needs a quick fix - the `testutil_copy_data()` function is not given an absolute path.

Reply:
thanks, fixed here: [0d3cc6d](https://github.com/wiredtiger/wiredtiger/pull/11896/commits/0d3cc6d2f114bb653019703d6cb3fd011051d1bb)